integration-tests/chaos/automation_chaos_test.go

```diff
@@ -125,6 +125,7 @@ func TestAutomationChaos(t *testing.T) {
 	l := logging.GetTestLogger(t)
 
 	registryVersions := map[string]eth_contracts.KeeperRegistryVersion{
+		// TODO is this correct? Action used to to configure OCR doesn't support registry 2.0 ¯\_(ツ)_/¯
```

Comment:
@anirudhwarrier  can you confirm?

Reply:
[SetConfig](https://github.com/smartcontractkit/chainlink/blob/daaf9d340ee0ec8e96f208c5c35a2899619269c5/integration-tests/contracts/ethereum_contracts_automation_seth.go#L208) works for v1.x and v2.0
[SetConfigTypeSafe](https://github.com/smartcontractkit/chainlink/blob/daaf9d340ee0ec8e96f208c5c35a2899619269c5/integration-tests/contracts/ethereum_contracts_automation_seth.go#L178) works for v2.1 and v2.2
And this part is done [here](https://github.com/smartcontractkit/chainlink/blob/da267d876d9eae3974c2de464a4f85c6eedbe691/integration-tests/chaos/automation_chaos_test.go#L284-L288)

@Tofel did you mean anything else than the above?

Reply:
I think you meant [CreateOCRKeeperJobs](https://github.com/smartcontractkit/chainlink/blob/00e17d8454ab9fd23397e974a75c79f11a8e466a/integration-tests/actions/automation_ocr_helpers.go#L177)

Reply:
yes, that action you mention looks like this:
```
	var contractVersion string
	if registryVersion == ethereum.RegistryVersion_2_2 {
		contractVersion = "v2.1+"
	} else if registryVersion == ethereum.RegistryVersion_2_1 {
		contractVersion = "v2.1"
	} else {
		require.FailNow(t, fmt.Sprintf("v2.1, and v2.2 are the only supported versions, but got something else: %v (iota)", registryVersion))
	}
```